src/wamr/env.cpp

```diff
@@ -0,0 +1,97 @@
+#include <faabric/util/logging.h>
+#include <storage/FileDescriptor.h>
+#include <wamr/WAMRWasmModule.h>
+#include <wamr/native.h>
+
+#include <stdexcept>
+
+// WAMR includes
```

Comment:
I don't think we need these `WAMR includes` comments on all the includes. We don't do it for any other libraries. It's sufficient to just put them after a newline.